cmake/SpectreSetupPythonPackage.cmake

```diff
@@ -44,11 +49,11 @@ configure_file(
   "${CMAKE_BINARY_DIR}/tmp/LoadPython.sh"
   "${CMAKE_BINARY_DIR}/bin/LoadPython.sh")
 
-# Copy the Python package to CMake's install directory
+# Install the Python package to the user-specified location.
```

Comment:
sorry, what's "the python package"? Do you mean "the SpECTRE python package"? I don't understand if this is copying python _itself_ or just our python code...

Reply:
Just our Python code, edited the fixup